extension/detect.js

```diff
@@ -0,0 +1,3 @@
+const isInstalledNode = document.createElement('div');
+isInstalledNode.id = 'apiary-browser-extension';
+document.body.appendChild(isInstalledNode);
```

Comment:
Without looking at how `content_scripts` works: any chance it could get executed before `body` is ready? Or is it added after DOM loaded or similar event?